### PR TITLE
EZP-31185: Restyling grid view in sub items

### PR DESCRIPTION
**Pull request by OstafinL**
| Question      | Answer
| ------------- | ---
| Tickets       | [EZP-31185](https://jira.ez.no/browse/EZP-31185)
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Restyling grid view in sub items. 
Added styles v3 to grid view in sub items. Changes include (single item):
- border size and border color
- item size
- font size and font color  

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: Resources/public/scss/modules/sub-items-list/_grid.view.item.scss [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
     cursor: pointer;
+    color: $ez-black;
     text-decoration: none;
     transition: background 0.3s $ez-admin-transition;
-    position: relative;
-    background: $ez-ground-base;
-    border-radius: calculateRem(4px);
-    margin-bottom: calculateRem(8px);
+    background: $ez-white;
+    box-shadow: 0px calculateRem(2px) calculateRem(10px) calculateRem(-4px) rgba(0,0,0,0.15);
```
Comment (dew326) [inline Resources/public/scss/modules/sub-items-list/_grid.view.item.scss]: When it's `0` we don't use units

File: Resources/public/scss/modules/sub-items-list/_grid.view.item.scss [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
+    border: 1px solid transparent;
```
Comment (dew326) [inline Resources/public/scss/modules/sub-items-list/_grid.view.item.scss]: `calculateRem(1px)`

File: Resources/public/scss/modules/sub-items-list/_grid.view.item.scss [hunk 3]
```diff
@@ -1,16 +1,22 @@
...
 
     &:hover,
     &:focus {
+        color: $ez-white;
         text-decoration: none;
-        background: #e5e3e3;
+        background: $ez-secondary-ground;
+        border: 1px solid $ez-secondary-ground;
```
Comment (dew326) [inline Resources/public/scss/modules/sub-items-list/_grid.view.item.scss]: `calculateRem(1px)`

File: Resources/public/scss/modules/sub-items-list/_grid.view.scss [hunk 4]
```diff
@@ -5,8 +5,8 @@
     padding-top: calculateRem(8px);
 
     .c-grid-view-item {
-        flex: 1 1 calc(100% / 6);
+        flex: 1 1 calc(100% / 5);
         max-width: calculateRem(166px);
-        margin-right: calculateRem(8px);
+        margin-right: calculateRem(35.75px);
```
Comment (dew326) [inline Resources/public/scss/modules/sub-items-list/_grid.view.scss]: This value is very big and strange. Could you explain why it has to be `35.75px`?

**Review by dew326 — CHANGES_REQUESTED** [review]
The PR title and commit message is missing the ticket number, also the name is not very accurate. 
I would go for the ticket name or something like `Restyled grid view in sub-items`.

Also what about long content names? In the wireframe it has `...` after the second line.

File: Resources/public/scss/modules/sub-items-list/_grid.view.item.scss [hunk 5]
```diff
@@ -50,12 +55,14 @@
...
-        font-size: calculateRem(14px);
-        padding: calculateRem(4px) calculateRem(8px);
-        color: $ez-black;
-        font-weight: bold;
+        height: calculateRem(33px);
+        display: -webkit-box;
+        -webkit-line-clamp: 2;
+        -webkit-box-orient: vertical;  
+        overflow: hidden;
+
```
Comment (GrabowskiM) [inline Resources/public/scss/modules/sub-items-list/_grid.view.item.scss]: Why empty line here?

Comment (dew326) [inline Resources/public/scss/modules/sub-items-list/_grid.view.item.scss]:
```suggestion
```

File: Resources/public/scss/modules/sub-items-list/_grid.view.scss [hunk 6]
```diff
@@ -5,8 +5,8 @@
     padding-top: calculateRem(8px);
 
     .c-grid-view-item {
-        flex: 1 1 calc(100% / 6);
+        flex: 1 1 calc(100% / 5);
         max-width: calculateRem(166px);
-        margin-right: calculateRem(8px);
+        margin-right: calculateRem(36px);
```
Comment (dew326) [inline Resources/public/scss/modules/sub-items-list/_grid.view.scss]: As agreed in the private discussion we will change this to 20px

**Review by dew326 — APPROVED** [review]

**Review by GrabowskiM — APPROVED** [review]

**Review by katarzynazawada — CHANGES_REQUESTED** [review]
On Edge browser border is not visible when element is not focused 
Screenshot 2019-12-12 at 13 59 07

**Review by GrabowskiM — APPROVED** [review]

**Review by dew326 — APPROVED** [review]

**Review by GrabowskiM — APPROVED** [review]

**Review by katarzynazawada — APPROVED** [review]
Looks good !

**Pull request merged**